test/images/nonewprivs/nnp.c

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Kubernetes Authors.
```

Comment:
Is there any specif reason my we are using C code? I see even os library in go has `Geteuid()` which can be used.